web/pdf_find_controller.js

```diff
@@ -386,6 +395,15 @@ var PDFFindController = (function PDFFindControllerClosure() {
       }
     },
 
+    updateUIResultCount:
+        function PDFFindController_updateUICountMatches() {
```

Comment:
Nit: The function in `pdf_find_bar.js` is called `updateResultsCount`, so for consistency we might want to add an "s" to the current method name, i.e. `updateUIResultsCount` instead!?.
(Also, the name after `function` isn't entirely consistent.)